README.md

```diff
@@ -58,3 +61,6 @@ This will:
 - Interactively ask for a new version number (and create a commit for the
   version change in package.json)
 - Publish the new release to NPM
+- Push a single commit to your local version of this repository
```

Comment:
How about updating this to something slightly more explicit?

```suggestion
- Push the newly created version bump commit to GitHub
```

Alternatively, I wonder if instead of explaining this as another manual step, maybe we should push automatically as part of running `yarn publishToNPM`? Maybe let's update the instruction for now and mull over automating it (maybe there's a reason not to instantly push 🤔)

Reply:
That does make sense, I'll update the wording for now to be a bit clearer, and we can look into automatic pushing soon.